### PR TITLE
[bitnami/elasticsearch] Add the possibility to provide extra argumets to the exporter (#5987)

### PR DESCRIPTION
**Pull request by lorisepita**
**Description of the change**

Add a `metrics.extraArgs` variable to add extra arguments to the exporter in order to enable metrics.

**Benefits**

- It is the now possible to enable all metrics

**Applicable issues**

  - Implements #5987

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

File: bitnami/elasticsearch/README.md [hunk 1]
```diff
@@ -281,6 +281,7 @@ The following table lists the configurable parameters of the Elasticsearch chart
 | `curator.envFromSecrets.*.from.key`               | - `secretKeyRef.key` used for environment variable                                                                                                                |                                                              |
 | `metrics.enabled`                                 | Enable prometheus exporter                                                                                                                                        | `false`                                                      |
 | `metrics.name`                                    | Metrics pod name                                                                                                                                                  | `metrics`                                                    |
+| `metrics.extraArgs`                               | Extra arguments to add to the default exporter command                                                                                                            | `{}`                                                         |
```
Comment (juan131) [inline bitnami/elasticsearch/README.md:284]: This parameter is an array. Therefore, the default value should be: `[]`:
```suggestion
| `metrics.extraArgs`                               | Extra arguments to add to the default exporter command                                                                                                            | `[]`                                                         |
```

File: bitnami/elasticsearch/values.yaml [hunk 2]
```diff
@@ -1082,6 +1082,13 @@ metrics:
     # pullSecrets:
     #   - myRegistryKeySecretName
 
+  ## Extra arguments for the exporter
+  ## ref: https://github.com/justwatchcom/elasticsearch_exporter
+  ##
+  # extraArgs:
+  #   - --es.snapshots
+  #   - --es.indices
```
Comment (juan131) [inline bitnami/elasticsearch/values.yaml:1090]: Use the example in the comments, and set a empty array by default value:
```suggestion
  ## Extra arguments for the exporter
  ## ref: https://github.com/justwatchcom/elasticsearch_exporter
  ## e.g
  ## extraArgs:
  ##   - --es.snapshots
  ##   - --es.indices
  ##
  extraArgs: []
```

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks so much for this contribution! Please check my comments when you have a chance.

**Review by juan131 — APPROVED** [review]
Thanks for addressing the changes! LGTM

**Comment by lorisepita** [comment]
Hi,
I applied the requested changes ! Tell me if anything alse needs to be changed :)

**Pull request merged**